css/properties/contain.json

```diff
@@ -11,24 +11,15 @@
             },
             "chrome_android": "mirror",
             "edge": "mirror",
-            "firefox": [
-              {
-                "version_added": "103"
-              },
-              {
-                "version_added": "69",
-                "partial_implementation": true,
-                "notes": "Firefox does not support the <code>style</code> value."
-              }
-            ],
+            "firefox": {
+              "version_added": "69"
+            },
             "firefox_android": "mirror",
             "ie": {
               "version_added": false
             },
             "oculus": "mirror",
-            "opera": {
-              "version_added": "40"
-            },
+            "opera": "mirror",
```

Comment:
Note, 39 is the matching version with chrome, and testing shows this is indeed the version that the options start working in.